src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlDataContractSerializerInputFormatter.cs

```diff
@@ -101,12 +121,23 @@ protected virtual XmlReader CreateXmlReader([NotNull] Stream readStream)
         }
 
         /// <summary>
-        /// Called during deserialization to get the <see cref="XmlObjectSerializer"/>.
+        /// Gets the type of the object to be serialized.
```

Comment:
Oh yes, copied this verbatim from XmlDataContractSerializerOutputFormatter.  It should instead say: "Gets the type to which the XML will be deserialized."